daniel-kavdi/public/scripts/controllers/articleController.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var app = app || {};
+
+(function(module) {
+  const articleController = {};
+  // DONE: Setup a function that kicks off the fetching and rendering of articles, using the same
+  // code that used to be in index.html.
+  // Also be sure to hide all the main section elements, and reveal the #articles section:
+  app.Article.fetchAll(app.articleView.initIndexPage);
```

Comment:
This should be _inside_ the function that gets invoked by `page` in your routes.